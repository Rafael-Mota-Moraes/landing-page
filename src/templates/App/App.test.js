import { screen } from "@testing-library/react";
import Home from "./index";
import { renderTheme } from "../../styles/render-theme";

describe("<Home />", () => {
  it("should render home", () => {
    const { debug } = renderTheme(<Home />);
  });
});
