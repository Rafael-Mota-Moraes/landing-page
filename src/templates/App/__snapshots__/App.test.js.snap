// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  class="sc-bcXHqe hMwhAx"
>
  <h1>
    Hello World!
  </h1>
</div>
`;
